src/handlers.rs

```diff
@@ -132,122 +172,101 @@ fn handle_start(
 }
 
 fn create_msg(
+  cmd_id: u32,
   builder: &mut FlatBufferBuilder,
-  args: &msg::BaseArgs,
-) -> deno_buf {
+  mut args: msg::BaseArgs,
+) -> Buf {
+  args.cmd_id = cmd_id;
   let base = msg::Base::create(builder, &args);
   msg::finish_base_buffer(builder, base);
   let data = builder.finished_data();
-  deno_buf {
-    // TODO(ry)
-    // The deno_buf / ImportBuf / ExportBuf semantics should be such that we do not need to yield
```

Comment:
This is gone. Does that mean you are okay now with yielding ownership?

Reply:
Yes, I think there's no way around it. Otherwise readFileSync will have to be implemented as a series of read calls that buffer up data.